### PR TITLE
W2 initialise fields.1

### PR DESCRIPTION
**Pull request by irvcaza**

### CODE REVIEW

File: worksheet2/main.c [hunk 1]
```diff
@@ -10,10 +10,23 @@
...
-    double tau, velocityWall;
+    double tau, velocityWall[3];    
 
-    readParameters(&xlength, &tau, &velocityWall, &timesteps, &timestepsPerPlotting, argc, argv);
+    readParameters(&xlength, &tau, velocityWall, &timesteps, &timestepsPerPlotting, argc, argv);
 
+    /* TODO: Add errors to failed allocation */
+    double  *collideField = (double *)  malloc((size_t)( 19*(xlength+2)*(xlength+2)*(xlength+2) ) * sizeof( double ));
+    double  *streamField = (double *)  malloc((size_t)( 19*(xlength+2)*(xlength+2)*(xlength+2) ) * sizeof( double ));
+    int  *flagField = (int *)  malloc((size_t)( (xlength+2)*(xlength+2)*(xlength+2) ) * sizeof( int ));
```
Comment (nasay) [inline worksheet2/main.c:20]: `warning: ISO C90 forbids mixed declarations and code`

File: worksheet2/initLB.c [hunk 2]
```diff
@@ -33,8 +34,50 @@ int readParameters(
     return 0;
 }
 
+void initialiseCell(double *collideField, double *streamField, int *flagField, int length_tot, int x, int y, int z, int flag){	
+
+	flagField[z * length_tot * length_tot + y * length_tot + x] = flag;
+
+	for (int i = 0; i < 19; ++i){
```
Comment (nasay) [inline worksheet2/initLB.c]: `error: ‘for’ loop initial declarations are only allowed in C99 mode`

Comment (nasay) [inline worksheet2/initLB.c]: By the way, sublime text also has to have an option, that tabs will be automatically changed to an equivalent number of spaces:)

**Comment by nasay** [comment]
you also can just make a first commit with meaningful (or you can change this message later) and then do fixups and then squash them into one commit: https://robots.thoughtbot.com/autosquashing-git-commits

**Comment by nasay** [comment]
https://github.com/nathanwbrei/cfdlab/issues/19

**Comment by nasay** [comment]
https://github.com/nathanwbrei/cfdlab/issues/16

File: worksheet2/main.c (hunk 1 above)
Comment (irvcaza) [inline worksheet2/main.c:20]: It didn't launched me the error, are you using other flags in the makefile?

**Pull request closed without merging**